### ISSUE TITLE
bun doesn't work to create StdioServerTransport

### ISSUE THREAD

**Issue by CryogenicPlanet** [modelcontextprotocol/typescript-sdk#91]
no errors are shown on either run the server or inside claude but no tools are detected - in the MCP hackathon rn so will give detailed reproduction steps in a bit but wanted to make a note of this

**Comment by axefrog** [comment]
In the tutorial:

```ts
async function main() {
  const transport = new StdioServerTransport();
  await server.connect(transport);
  console.error("Weather MCP Server running on stdio");
}
```

Running from the command line shows an immediate fallthrough to the `console.error` line. No errors reported by the call to `server.connect`.

**Comment by jspahrsummers** [comment]
I'm afraid I'm not very familiar with bun. Do you have a reproducible example project?

**Comment by KKonstantinov** [comment]
Bun will require significant refactoring and breaking changes in order to be fully supported, please follow https://github.com/modelcontextprotocol/typescript-sdk/issues/809 - it's one of the items on the list.
